assets/scss/components/_card.scss

```diff
@@ -0,0 +1,149 @@
+.resource-block {
+  .resource-block-left-large {
+    width: 9.375rem;
+    height: 9.375rem;
+    margin-top: 2.5rem;
+    background: #1d9ad1;
+    transition: all 1s ease;
+    z-index: 9;
+  }
+
+  .resource-block-left-stripes {
+    display: block;
+    width: 8.75rem;
+    height: 8.75rem;
+    margin-top: 7.5rem;
+    margin-left: -3.75rem;
+    background: repeating-linear-gradient(
+      -45deg,
+      #1d9ad1,
+      #ffffff 2px,
+      #ffffff 13px,
+      #1d9ad1 13px
+    );
+    transition: all 1s ease;
+    opacity: 0.5;
+  }
+}
+
+.resource-block-item {
+  flex: 1 0 100%;
+  max-width: 100%;
+  margin-bottom: 2.5rem;
+  &img {
```

Comment:
@louisefindlay23 looks like this `img` block styling is not being applied due to syntax error. You can remove the `&` symbol here as nesting a style block does not require the `&` symbol. 

The `&` symbol would be used if this `img` tag also had a class name of `.resource-block-item`. In this case the `img` tag is nested inside that element, so we can remove the `&` symbol. 

Also to make the image closer to the sizing on the Section blog, can you change the height to `12rem` on this `img` styling?

Reply:
This PR should be good after this last update. Let me know when you have completed this update and I'll do one more review and likely merge this PR. Thanks for all you hard work here @louisefindlay23!!

Reply:
Thanks for your invaluable feedback, @hborrelli1. 😊 That's the changes implemented. 🚀 Quite new to SCSS so thanks for helping me with the syntax. Glad to be of help. 

Will the pinned articles feature, that Hector mentioned, be a future CCC feature request? Would like to express my interest if you'll be looking for CCs for this in the future.

Reply:
@louisefindlay23 it may be we have to look at the EngEd product queue 1st to see where that lines up. 

Once we have the issue filled out we will be sure to ping the team 💯